firmware_mod/config/userscripts/startup/motor_calibrate.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+. /system/sdcard/scripts/common_functions.sh
+
+## Motor Calibrate 
+motor reset_pos_count
```

Comment:
I am not sure we should reset the position upon startup. It should be saved somewhere and recovered upon startup.

Reply:
I tried to do that, but i can´t complete the code. 
So i prefer at least reset the position to give me the correct X and Y position

Reply:
I'm actually +1 for motor calibration on startup. If you take a look at most Dafang or Mi cameras – they do calibration on startup. Without this my camera records wall after power down. :p

Saving position and restoring it might be ok, but as time passes – values can drift from perfect. Power down would be a nice moment to reset it.